### PR TITLE
Create metrics.html.md.erb

### PR DESCRIPTION
**Pull request by sfawcett123**
Initial Document,

### CODE REVIEW

File: source/documentation/standards/metrics.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+last_reviewed_on: 2021-03-15
+review_in: 3 months
+---
+
+# <%= current_page.data.title %>
+
+## User needs
+
+Users should be able to label metric data so that it is compatible with
+UK Government PaaS Exporter metrics and common across applications.
```
Comment (saliceti) [inline source/documentation/standards/metrics.html.md.erb:12]: Should we list the metrics reserved by paas?

File: source/documentation/standards/metrics.html.md.erb [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+- **HELP**  Description of the Metric
+- **TYPE**  Should be one of the [Prometheus Metric Types](https://prometheus.io/docs/concepts/metric_types/)
+- **Metric Name** The name of the metric should identify what it contains. 
+- **Label Name** labels are used to differentiate the characteristics of the thing that is being measured
+  
+Where possible use the same metric name and label name across different applications. However, ensure that the value is consistent in scale and the data it
+represents.
+
+```
+cpu_usage_percentage{ app="get-into-teaching-api" }  75
```
Comment (saliceti) [inline source/documentation/standards/metrics.html.md.erb:29]: This looks like a metric provided by paas. Could we list a more custom one for the example?

File: source/index.html.md.erb [hunk 3]
```diff
@@ -25,6 +25,7 @@ It is inspired by the [GDS Way](https://gds-way.cloudapps.digital) and the
 ### Building software
 
 * [Licensing software or code](documentation/standards/licencing-software-or-code.html)
+* [Metrics](documentation/standards/metrics.html)
```
Comment (saliceti) [inline source/index.html.md.erb:28]: I think it should be a subsection of paas as it's quite specific to paas. We only use prometheus in paas I believe.

File: source/documentation/standards/metrics.html.md.erb (hunk 1 above)
Comment (sfawcett123) [reply source/documentation/standards/metrics.html.md.erb:12]: No, but I will put a link to https://github.com/alphagov/paas-prometheus-exporter

**Comment by saliceti** [comment]
[Guidance on logging](https://trello.com/c/GrD6oP7p/216-guidance-on-logging)

**Comment by saliceti** [comment]
Needs rework as the docs structure has changed

**Pull request closed without merging**